Mathlib/LinearAlgebra/BilinearMap.lean

```diff
@@ -204,6 +204,46 @@ theorem domRestrict₁₂_apply (f : M →ₛₗ[ρ₁₂] N →ₛₗ[σ₁₂]
     (x : p) (y : q) : f.domRestrict₁₂ p q x y = f x y := rfl
 #align linear_map.dom_restrict₁₂_apply LinearMap.domRestrict₁₂_apply
 
+section restrictScalars
+
+variable (R' S' : Type _)
+
+variable [Semiring R'] [Semiring S'] [Module R' M] [Module S' N] [Module R' Pₗ] [Module S' Pₗ]
```

Comment:
My understanding was that the `M` and `N` variables were only supposed to be used for semilinear maps. Can you add `ₗ` suffices?

Reply:
No, `M` and `N` are generally used, the subscript `l` is for cases where you need less assumptions, but that is not true here.

Reply:
I didn't understand that from the comment on line 37:
> -- the `ₗ` subscript variables are for special cases about linear (as opposed to semilinear) maps

On the other hand, that seems to be the general approach since leanprover-community/mathlib#10373 e.g. the definition of `mk₂'`. So this is okay with me.